### ISSUE TITLE
Date Picker on Explorer page is not keyboard navigable

### ISSUE THREAD

**Issue by micahchiang** [odota/web#2858]
**Current Behavior**
When users are tabbing through the form on the Explorer page, getting to minDate and maxDate loses outline. The focus event is fired, but the datepicker doesn't open when using the keyboard. Focus is regained when getting to the `Order` field.

**Expected behavior/code**
Users should be able to open the datepicker and select min/max date with the keyboard only. 

**Console Output**
No console output as this occurs in the browser and doesn't have a corresponding output in devtools. 

**Environment**
- Browser Version: Chrome v92
- OS: OSX 11.4

**Possible Solution**
The datepicker component is imported from material-ui. The packages we use are fairly outdated, and updating them may resolve this issue.
